pkg/aggregator/apiserver/plugin/admission/admission.go

```diff
@@ -0,0 +1,78 @@
+package admission
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	admissionv1 "k8s.io/api/admission/v1"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+func ToAdmissionRequest(a *admissionv1.AdmissionReview) (*backend.AdmissionRequest, error) {
+	if a.Request == nil {
+		return nil, errors.New("admission review request is nil")
+	}
+	op, err := ToAdmissionOperation(a.Request.Operation)
+	if err != nil {
+		return nil, err
+	}
+
+	return &backend.AdmissionRequest{
+		PluginContext: backend.PluginContext{},
+		Operation:     op,
+		Kind: backend.GroupVersionKind{
+			Group:   a.Request.Kind.Group,
+			Version: a.Request.Kind.Version,
+			Kind:    a.Request.Kind.Kind,
+		},
+		ObjectBytes:    a.Request.Object.Raw,
+		OldObjectBytes: a.Request.OldObject.Raw,
+	}, nil
+}
+
+func ToAdmissionOperation(o admissionv1.Operation) (backend.AdmissionRequestOperation, error) {
+	switch o {
+	case admissionv1.Create:
+		return backend.AdmissionRequestCreate, nil
+	case admissionv1.Delete:
+		return backend.AdmissionRequestDelete, nil
+	case admissionv1.Update:
+		return backend.AdmissionRequestUpdate, nil
+	case admissionv1.Connect:
+		// TODO: CONNECT is missing from the plugin SDK
+		return 3, nil
```

Comment:
what is 3, and what would CONNECT mean?  would this be a sub-resource request?

Reply:
> would this be a sub-resource request

yeah, specifically using the `CONNECT` verb. so this wouldn't apply to something like the `status` subresource. i didn't know it was possible, but it seems like it won't hurt to handle it just in case it's needed in the future. thoughts?